packages/auth-helpers/src/lib/siwe/siwe-helper.ts

```diff
@@ -0,0 +1,98 @@
+import { SiweMessage } from 'siwe';
+
+import {
+  CapacityDelegationFields,
+  CapacityDelegationRequest,
+  ILitNodeClient,
+  LitResourceAbilityRequest,
+} from '@lit-protocol/types';
+
+/**
+ * Sanitizes a SIWE message by unescaping double-escaped newlines and replacing escaped double quotes with single quotes.
+ *
+ * @param message - The SIWE message to sanitize.
+ * @returns The sanitized SIWE message.
+ */
+export function sanitizeSiweMessage(message: string): string {
```

Comment:
Why do we need to do this? Have you tried without the sanitization? What error do you get without it?

Reply:
`///` in the siwe message. Howard added this 10 months ago you might want to ask him instead.